Main/Display/Gpu.cs

```diff
@@ -241,7 +241,9 @@ unsafe void Gpu_Paint(object sender, PaintEventArgs e)
 
             e.Graphics.CompositingMode = System.Drawing.Drawing2D.CompositingMode.SourceCopy;
             e.Graphics.CompositingQuality = System.Drawing.Drawing2D.CompositingQuality.HighSpeed;
-            e.Graphics.InterpolationMode = System.Drawing.Drawing2D.InterpolationMode.NearestNeighbor;
+            // Bilinear interpolation has effect very similar to real HW 
+            //e.Graphics.InterpolationMode = System.Drawing.Drawing2D.InterpolationMode.NearestNeighbor;
+            e.Graphics.InterpolationMode = System.Drawing.Drawing2D.InterpolationMode.Bilinear;
```

Comment:
The only problem with this, is that the Frames Per Second (FPS) drops to about 12, on my machine, from 60.
So, performance of this InterpolationMode is not very efficient.